src/webots/nodes/WbBackground.cpp

```diff
@@ -66,15 +66,15 @@ static int gCoordinateSystemSwap(int i) {
   static const int enu_swap[] = {5, 4, 0, 1, 3, 2};
   if (WbWorld::instance()->worldInfo()->coordinateSystem() == "ENU")
     return enu_swap[i];
-  else
+  else  // "NUE" or "EUN"
```

Comment:
I am not sure in the EUN case it makes sense to rotate the background.
In the ENU case it clearly makes sense since the sky should be up, but here the sky is already up, and in the background, there is no notion of north and east but rather front and right. Furthermore, if we want to keep 100% compatibility with < R2020b worlds with northDirection set to '0 0 1' we should not rotate it. 

@cyberbotics/maintainers what do you think?